readme.md

```diff
@@ -479,3 +479,7 @@ When set to `true` the filter logic is inverted, i.e. requests originating from
 ### Countries `countries`
 
 A list of country codes from which connections to the service should be allowed. Logic can be inverted by using the [`blackListMode`](#back-list-mode-blacklistmode).
+
+### Allowed IP addresses `allowedIPAddresses`
+
+A list of explicitly allowed IP addresses. IP addresses added to this list will always be allowed.
```

Comment:
Maybe add some examples too?

Supports both single IP-addresses and CIDR-notated subnets.  
```
allowedIPAddresses:
- 192.0.2.10
- 203.0.113.0/24
- 2001:db8:1234:/48

Reply:
Good point. I added the description.